.vscode/launch.template.json

```diff
@@ -49,8 +49,8 @@
         "-t",
         "${input:testName}",
         "--pool",
-        "threads",
-        "--poolOptions.threads.singleThread"
+        "forks",
```

Comment:
since merging https://github.com/ChainSafe/lodestar/pull/7485 it's no longer possible to use thread to run most tests, @wemeetagain based on the PR there shouldn't be much of a difference but maybe we should dig into what changed in napi-rs bindings that's causing these segfaults, not sure yet if it's the right place to investigate but is it possible that the napi-rs version changed when you updated `@chainsafe/pubkey-index-map` from 2.x to 3.x?

Reply:
seems like napi-rs 3 is still in alpha, we are using [3.0.0-alpha.28](https://github.com/ChainSafe/pubkey-index-map/blob/09cf357940e43742bcb29c1c35472d82a8aa52cb/Cargo.toml#L11) but there are 5 more pre-releases with some PRs may change something for us, although can't really tell.

but we can easily reproduce the segfault by changing `forks` to `threads` here
https://github.com/ChainSafe/lodestar/blob/03bfb04361a03979eee490d0727ae13c26644bd9/configs/vitest.config.unit.ts#L20

I guess problem is we would need to do it for async compute package as well, not just pubkey index map